src/main/java/com/example/naitei19javaecommerce/controller/user/CartController.java

```diff
@@ -0,0 +1,29 @@
+package com.example.naitei19javaecommerce.controller.user;
+
+import com.example.naitei19javaecommerce.model.CartItem;
+import com.example.naitei19javaecommerce.model.Product;
+import com.example.naitei19javaecommerce.service.Impl.CartItemServiceImpl;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Controller
+public class CartController {
+    @Autowired
+    private CartItemServiceImpl cartItemServiceImpl;
```

Comment:
```suggestion
    @Autowired
    private CartItemService cartItemServiceImpl;
```
ri chứ, chứ em dùng class thì cái interface đó có ý nghĩa j đâu cu